pkgs/development/compilers/llvm/11/compiler-rt/default.nix

```diff
@@ -13,21 +13,18 @@ stdenv.mkDerivation rec {
   inherit version;
   src = fetch pname "0x1j8ngf1zj63wlnns9vlibafq48qcm72p4jpaxkmkb4qw0grwfy";
 
-  nativeBuildInputs = [ cmake python3 llvm ];
+  nativeBuildInputs = [ cmake python3 llvm.dev ];
   buildInputs = lib.optional stdenv.hostPlatform.isDarwin libcxxabi;
 
   NIX_CFLAGS_COMPILE = [
     "-DSCUDO_DEFAULT_OPTIONS=DeleteSizeMismatch=0:DeallocationTypeMismatch=0"
   ];
 
   cmakeFlags = [
+    "-DCOMPILER_RT_OS_DIR="
```

Comment:
IIUC, this causes `libclang_rt.osx.a` to be installed to `$out/lib`, instead `$out/lib/darwin` as it was previously, which means it can no longer be found by the linker driver, which always uses `darwin` (see [MachO::AddLinkRuntimeLib](https://github.com/llvm/llvm-project/blob/1fdec59bffc11ae37eb51a1b9869f0696bfd5312/clang/lib/Driver/ToolChains/Darwin.cpp#L1070-L1075)). I noticed it trying to cross compile `m4` for `aarch64-darwin`, but I don't yet know how it works on `x86_64-darwin`.

Previously
```
result
└── lib
    ├── darwin
    │   ├── libclang_rt.cc_kext.a
    │   └── libclang_rt.osx.a
    ├── libclang_rt.cc_kext.a -> darwin/libclang_rt.cc_kext.a
    └── libclang_rt.osx.a -> darwin/libclang_rt.osx.a
```

Now
```
result
└── lib
    ├── libclang_rt.cc_kext.a
    └── libclang_rt.osx.a
```

Reply:
@thefloweringash sorry about that. Does this effect the current 7-based bootstrapping or just the new one?